packages/dev-server-rollup/src/rollupAdapter.ts

```diff
@@ -172,7 +172,7 @@ export function rollupAdapter(
             !['/', './', '../'].some(prefix => resolvableImport.startsWith(prefix)) &&
             adapterOptions.throwOnUnresolvedImport
           ) {
-            const errorMessage = red(`Could not resolve import ${cyanBright(`"${source}"`)}.`);
+            const errorMessage = red(`Could not resolve import ${cyan(`"${source}"`)}.`);
```

Comment:
cyan !== cyanBright ?

Reply:
Yes it's not quite the same. See available colors [here](https://github.com/ai/nanocolors/blob/44908c1348e3137f6ede3b80bac74766b6a58244/example.png). Any better ideas?

Reply:
![Screenshot 2021-09-22 at 18 45 22](https://user-images.githubusercontent.com/24378/134386545-bc39729b-48f6-423d-b0de-55b0caa2984a.png)

ok the difference is not that big...


once thing I could not find is... can I disable the colors without a system variable? 🤔

```js
import { options } from 'colorette';
describe('cli', () => {
  before(() => {
    // ignore colors in tests as most CIs won't support it
    options.enabled = false;
  });
```

Reply:
Functions will use colors only if Nano Colors auto-detect that current environment supports colors.
There is an option to override color support auto detection, [example](https://github.com/ai/nanocolors/tree/44908c1348e3137f6ede3b80bac74766b6a58244#conditional-support).

Reply:
sorry for being picky 😅
change looks good.

@LarsDenBakker you ok with the color change? you picked those initially I think

Reply:
Fine with me